Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM osrf/ros:jazzy-desktop-full
```

Comment:
do we need full? we're not doing any ROS GUI stuff so might be overkill but I guess it doesn't cost too much